### PR TITLE
fixing minor content issues in lesson 14

### PR DESCRIPTION
**Pull request by STFinancial**
### Before you submit, please check that:

- [ ] You labeled the PR with appropriate labels, if any.
- [ ] You mentioned a related issue if this PR related to it, e.g. `Fixes #22` or `Related #17`.
- [ ] You performed a self-review of your own code
- [ ] You commented on code, particularly in hard-to-understand areas
- [ ] You asked any particular reviewer to review. If you aren't sure, ask.
- [ ] Your PR adheres to our `CONTRIBUTION GUIDELINES`

After you've read this notice feel free to remove it.
✄ -----------------------------------------------------------------------------

**What does this PR do?**
A clear and concise description of what the PR aims to accomplish.

**Description of Task to be completed?**

**How should this be manually tested?**
A clear and concise description of how the changes can be tested.

**Any background context you want to provide?**

**Screenshots(optional)**
If applicable, add screenshots to help to explain the problem/solution.

**Additional context(optional)**
Add any other context if any.

### CODE REVIEW

File: lessons/introduction-to-smartpy/14/14.mdx [hunk 1]
```diff
@@ -102,11 +102,11 @@ editor:
 
 ### Intro: 
 
-You would be already be pretty familiar with basic decision making in Python using `if...else` statements.  
+You should already be pretty familiar with basic decision making in Python using `if...else` statements.
```
Comment (prastut) [inline lessons/introduction-to-smartpy/14/14.mdx:105]: @STFinancial "should" gives a strong connotation. I would let it "would be" remain as it is. 

Our principles for content design were (should be documented somewhere, thanks for bringing this up)
1. Take no assumptions for learner's state of mind and skills. 
2. Give as much affordance to the learner.

File: lessons/introduction-to-smartpy/14/14.mdx [hunk 2]
```diff
@@ -208,4 +208,4 @@ You can also tinker around with the final code on [SmartPy’s online IDE](https
 Read more in depth about: 
 
 1. Technical pov of using `sp.if` over `if` in your smart contract code - under reference manual's "Control & Syntactical Sugar" section [here.](https://smartpy.io/demo/reference.html#_control_and_syntactic_sugar)
-2. Why you cannot use `and`, `or`, `not` in your smart contract code and instead need to use `&`, `|`, `~` respectively - under reference manual's "Boolean" section [here.](https://smartpy.io/demo/reference.html#_booleans)
\ No newline at end of file
+2. Why you cannot use `and`, `or`, `not` in your smart contract code and instead need to use `&`, `|`, `~` respectively - under reference manual's "Boolean" section [here.](https://smartpy.io/demo/reference.html#_booleans)
```
Comment (prastut) [inline lessons/introduction-to-smartpy/14/14.mdx:211]: @STFinancial I am not able to detect any visible changes here through 2-3 reading. Can you help me in pointing to what was the change here?

Comment (STFinancial) [reply lessons/introduction-to-smartpy/14/14.mdx:211]: I don't know why this is being listed as a diff, I also did not make a change here

File: lessons/introduction-to-smartpy/14/14.mdx (hunk 1 above)
Comment (STFinancial) [reply lessons/introduction-to-smartpy/14/14.mdx:105]: That's fine, but "would be already be" says "be" twice and not grammatical, at least in my american english. Perhaps "you may already be pretty familiar" would be better instead?

Comment (prastut) [reply lessons/introduction-to-smartpy/14/14.mdx:105]: > That's fine, but "would be already be" says "be" twice and not grammatical
Agreed. 

> Perhaps "you may already be pretty familiar" would be better instead?
Agreed. Request to make this change and we can merge this 🚀

**Pull request closed without merging**